workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -16,6 +16,7 @@ public class GdkToolsConfigurationWindow : EditorWindow
         internal const string VerboseLoggingLabel = "Verbose logging";
         internal const string CodegenLogOutputDirLabel = "Log output directory";
         internal const string CodegenOutputDirLabel = "C# output directory";
+        internal const string CodegenEditorOutputDirLabel = "C# Editor output directory";
```

Comment:
As a drive-by, could update the label to make it clearer that this is for generated code

Reply:
These are rendered under the "Code Generation" header in the settings, what did you have in mind? 

![image](https://user-images.githubusercontent.com/13353733/83251917-443f6280-a1a2-11ea-832a-399f6729d469.png)

Reply:
ah yes I didn't expand to see the heading 

ignore this